packages/lesswrong/components/tagging/FooterTagList.tsx

```diff
@@ -269,32 +283,38 @@ const FooterTagList = ({
       >
         <div className={classNames(classes.frontpageOrPersonal, {
           [classes.smallText]: smallText,
+          [classes.noBackground]: noBackground,
+          [classes.neverCoreStyling]: neverCoreStyling,
         })}>
           {label}
         </div>
       </HoverOver>
     );
-  }, [classes, smallText]);
+  }, [classes, smallText, noBackground]);
 
   // Post type is either Curated, Frontpage, Personal, or uncategorized (in which case
   // we don't show any indicator). It's uncategorized if it's not frontpaged and doesn't
   // have reviewedByUserId set to anything.
   let postType = post.curatedDate
     ? <Link to={contentTypeInfo.curated.linkTarget} className={classes.postTypeLink}>
-        <PostTypeTag label="Curated" tooltipBody={contentTypeInfo.curated.tooltipBody}/>
+        <PostTypeTag label="Curated" tooltipBody={contentTypeInfo.curated.tooltipBody} neverCoreStyling/>
       </Link>
     : (post.frontpageDate
       ? <MaybeLink to={contentTypeInfo.frontpage.linkTarget} className={classes.postTypeLink}>
-          <PostTypeTag label="Frontpage" tooltipBody={contentTypeInfo.frontpage.tooltipBody}/>
+          <PostTypeTag label="Frontpage" tooltipBody={contentTypeInfo.frontpage.tooltipBody} neverCoreStyling/>
         </MaybeLink>
       : (post.reviewedByUserId
         ? <MaybeLink to={contentTypeInfo.personal.linkTarget} className={classes.postTypeLink}>
-          <PostTypeTag label="Personal Blog" tooltipBody={contentTypeInfo.personal.tooltipBody}/>
+            <PostTypeTag label="Personal Blog" tooltipBody={contentTypeInfo.personal.tooltipBody} neverCoreStyling/>
```

Comment:
```suggestion
            <PostTypeTag label="Personal Blog" tooltipBody={contentTypeInfo.personal.tooltipBody} neverCoreStyling={neverCoreStyling} />
```

I assume this is what you intended? (Plus the others above this)